### DISCUSSION TITLE
Please add support for half float as an input type

### DISCUSSION THREAD

**Discussion by gregbenz** [google/libultrahdr#143] [Ideas]
The demo app supports P010 and RGB1010102 inputs. It would be helpful if it would also support the halfFloat inputs (which is already supported for decoding).
https://github.com/google/libultrahdr/blob/3dccd6113bf392dbeb6389053d7e868f3dbb6cda/examples/ultrahdr_app.cpp#L1183

**Comment by ram-mohan** [comment]
We are making some more changes towards adding different sub sampling formats for hdr intent and sdr intent (444, 422, ...) we will try to add this along with them. Thank you.
